### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by Nikoh77** [emmercm/igir#870] [Ideas]
A discord server for igir users would be very useful, maybe Christian could create it.

**Comment by emmercm** [comment]
I completely missed your discussion, I apologize @Nikoh77.

I'll be honest, I'm not personally a big Discord user, and I'm not sure I would have a lot of time to commit to a server. As a result, I would worry about the moderation of community guidelines that would need to be established.

I personally like the centralization of discussions in GitHub, especially for searchability. But I'll admit it's a poor substitute for real-time communication, and GitHub notification settings are fairly buried under `Watch > Custom > Discussions`.

What I would ask is for people to comment on how they would be interested in using a Discord server, and what level of involvement they would like to have in it.

**Comment by Europia79** [comment]
Discord would be great to be able to quickly & easily help out new users via Voice Chat &/or Screen Share.  Plus, after Moderators have helped out a fair number of users, then it is possible that a pattern of misunderstanding could emerge that exposes a deficiency in the documentation and explain why users repeatedly "get it wrong":  And hence, be able to update & improve the documentation.

Also, Discord would be a great to serve as a kind of a "proving grounds" of "informal issues" where someone might be encountering a problem, but they're not sure if it's a problem in the software that they really need to report, OR, if they're just doing something wrong.

There are also other kinds of "issues", like suggestions, where someone might be hesitant on posting, because they're not sure if the idea is "too crazy" to suggest.  Or, maybe they have not fully formulated their idea(s) yet ?  For these two use-case scenarios, Discord could be a great setting for peer review & collaboration (to help them develop their ideas & suggestions before formally submitting them to Github).

Altho, me personally, I would not have a lot of time to devote to it either:

But I would definitely love join the **igir** Discord & jump on when I can !!!
